src/main/java/net/sigmalab/maven/plugin/jira/ReleaseVersionMojo.java

```diff
@@ -108,18 +109,15 @@ public Version calculateLatestReleaseVersion(Iterable<Version> versions) {
      * @return
      */
     public boolean isVersionAlreadyPresent(Version[] versions, String newDevVersion) {
-        boolean versionExists = false;
-
         if ( versions != null ) {
             for ( Version remoteVersion : versions ) {
                 if ( remoteVersion.getName().equalsIgnoreCase(newDevVersion) ) {
```

Comment:
The use of the `break;` statement here is a bit like a `goto` - not technically wrong, but not very nice either so I've restructured this appropriately.